core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -173,10 +177,11 @@ trait SubmissionComponent {
           .foldLeft(Map.empty[String, Int])(_ |+| _)
         val maybeWorkflowIds = states.getOrElse(submissionRec.id, Seq.empty).flatMap(_.workflowId).sorted
         val workflowIds = if (maybeWorkflowIds.nonEmpty) Some(maybeWorkflowIds) else None
-        SubmissionListResponse(unmarshalSubmission(submissionRec, config, entityRec.map(_.toReference), Seq.empty),
-                               workflowIds,
-                               statusCounts,
-                               config.deleted
+        SubmissionListResponse(
+          unmarshalSubmission(submissionRec, config, entityRec.map(_.toReference), Seq.empty, None), // TODO get these
```

Comment:
This is the spot where I was defeated by storing a list as a comma-separated string.